cup/src/pages/zero-one/organisms/gameItem/gameItem.tsx

```diff
@@ -0,0 +1,31 @@
+import { motion } from 'framer-motion'
+import styles from '../../styles.module.css'
```

Comment:
[must]
Don't use "../".
Please replace "../../styles.module.css" -> "@/pages/zero-one/styles.module.css"

Reply:
@lamb-sni 
thx. I have made the necessary corrections.